Magento_Customer/templates/form/resetforgottenpassword.phtml

```diff
@@ -0,0 +1,81 @@
+<?php
+/** @var \Magento\Customer\Block\Account\Resetpassword $block */
+?>
+<form
+    class="forgot-password"
+    action="<?= $block->escapeUrl($block->getUrl('*/*/resetpasswordpost', ['_query' => ['id' => $block->getCustomerId(), 'token' => $block->getResetPasswordLinkToken()]])) ?>"
+    method="post"
+    <?php if ($block->isAutocompleteDisabled()) :?> autocomplete="off"<?php endif; ?>
```

Comment:
change to `<?php if ($block->isAutocompleteDisabled()): ?>`

Reply:
Done